app/pages/index.js

```diff
@@ -2,41 +2,24 @@ import Head from "next/head";
 import Layout, { siteTitle } from "../components/layout";
 import utilStyles from "../styles/utils.module.css";
 import { getSortedPostsData } from "../lib/posts";
-import Link from "next/link";
-import Date from "../components/date";
+import ChaletEntrance from "../components/homepage/ChaletEntrance";
+import Mission from "../components/homepage/Mission";
+// import HomeSections from "../components/homepage/HomeSections";
```

Comment:
please do not commit commented code

if its commented its not needed, if its not needed should be committed

Reply:
okay, thank you for the feedback. This is commented because I pretend to create the file HomeSections in the future (was created in the next pull requests)